src/ZendeskApi.Contracts/Models/User.cs

```diff
@@ -92,5 +92,8 @@ public class User : IZendeskEntity
 
         [DataMember(Name = "ticket_restriction", EmitDefaultValue = false)]
         public string TicketRestriction { get; set; }
+
+        [DataMember(Name = "user_fields", EmitDefaultValue = false)]
+        public Dictionary<string, object> UserFields { get; set; }
```

Comment:
Can you make a custom model for user_fields, like we have with custom_fields? https://github.com/justeat/ZendeskApiClient/blob/master/src/ZendeskApi.Contracts/Models/CustomField.cs

I haven't done any work with user fields, so it might not be possible. However, if it is I'd like to maintain a level of consistency. I also think the explicit model approach makes it easier to understand use-cases for some of these more complex properties.